.changeset/remove-grounding.md

```diff
@@ -0,0 +1,4 @@
+---
+'@sap-ai-sdk/orchestration': minor
+---
+[Compatibility Note] Remove grounding module related types, which should not be used.
```

Comment:
[q] Do we really need a compatibility note? This was never meant to be part of the public api. The fact that we expose `ModuleConfigs` in `index.ts` is incorrect.

Reply:
For SDK user, I agree, it's not relevant, if they only use public APIs, so I deleted the compatibility notes.
I put it there, to make it transparent for those who noticed our grounding related code.

Reply:
will work on `ModuleConfigs` in a different PR.